Shared/Extensions/SplitView/SplitViewBehavior.cs

```diff
@@ -1,8 +1,14 @@
-﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__
+﻿#if WINDOWS_UWP || __ANDROID__ || __IOS__ || __WASM__ 
```

Comment:
Don't you need  || WINDOWS10_0_18362_0_OR_GREATER?

Reply:
@GabrielChouinardLetourneau this one is resolved but there's no information why

Reply:
I forgot to add the condition for Windows